wordpress-mobile/WordPress-iOS
# UL&S: Remove .showLoginMethods segue

This PR removes the .showLoginMethods segue and programmatically presents the 3 button view to the user.

Ref. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/271

### To test
The main focus is navigation. Both the 3 button view presenting and navigating the user to the next step after a button or link was selected.

1. Check out this branch
2. `rake dependencies`
3. Build and run on a device (recommended over sims b/c SIWA testing)
4. Log out if you are logged in
5. Select "Log In".  **Expected:** the 3 button view appears and dims the carousel
6. Select "Continue with WordPress.com". **Expected:** the user is navigated to the LoginEmailVC. Go back.
7. Log in > Continue with Google > Continue. **Expected:** the user is presented with a Google sign in web view. Go back.
8. Log in > Continue with Apple > Continue. **Expected:** the user is navigated (without crashing!) to their next expected screen - 2fa or "logged in as..." .

### Do not merge until
1. The Authenticator PR has been merged. Ref: wordpress-mobile/WordPressAuthenticator-iOS#271
2. A new Authenticator release has been created
3. The release is published on Cocoapods trunk
4. This PR points to the new release

PR submission checklist:
